lib/Service/RoomService.php

```diff
@@ -103,6 +109,7 @@ public function create(string $name, string $welcome, int $maxParticipants, bool
 		$room->setMediaCheck($mediaCheck);
 		$room->setCleanLayout(false);
 		$room->setJoinMuted(false);
+		$room->setLogoutURL($this->urlGenerator->getBaseUrl());
```

Comment:
This means we can not distinguish if we are using a user defined url or not in the future. Imagine an installation is moved to a new domain. All fields would need to be updated.

Please use the empty string here and check for it in the API class.

Reply:
It also means that currently there is no fallback for existing rooms.